### PR TITLE
compatibility with Kinetic

### PR DESCRIPTION
**Pull request by pietrocolombo**
Changing this logging line to retain compatibility with Kinetic. refs  #136

For retain compatibility with Kinetic
refs #136

### CODE REVIEW

File: piksi_multi_rtk_ros/src/piksi_multi_rtk_ros/piksi_multi.py [hunk 1]
```diff
@@ -1565,7 +1565,7 @@ def cb_sbp_imu_aux(self, msg_raw, **metadata):
         self.gyro_scale = gyro_range * PiksiMulti.kGyroPrescale
 
         self.has_imu_scale = True
-        rospy.loginfo_once("Received IMU scale.")
+        rospy.loginfo("Received IMU scale.")
```
Comment (rikba) [inline piksi_multi_rtk_ros/src/piksi_multi_rtk_ros/piksi_multi.py:1568]: Can you wrap the log info to make sure it is published only once?

```suggestion
        if not self.has_imu_scale:
            rospy.loginfo("Received IMU scale.")
        self.has_imu_scale = True  
```

**Review by rikba — CHANGES_REQUESTED** [review]
Thanks for the contribution. Please see the comment.

File: piksi_multi_rtk_ros/src/piksi_multi_rtk_ros/piksi_multi.py [hunk 2]
```diff
@@ -1565,7 +1565,9 @@ def cb_sbp_imu_aux(self, msg_raw, **metadata):
         self.gyro_scale = gyro_range * PiksiMulti.kGyroPrescale
 
         self.has_imu_scale = True
```
Comment (rikba) [inline piksi_multi_rtk_ros/src/piksi_multi_rtk_ros/piksi_multi.py:1567]: Please remove this line.

**Review by rikba — CHANGES_REQUESTED** [review]

File: piksi_multi_rtk_ros/src/piksi_multi_rtk_ros/piksi_multi.py (hunk 2 above)
Comment (pietrocolombo) [reply piksi_multi_rtk_ros/src/piksi_multi_rtk_ros/piksi_multi.py:1567]: I don't understand which line.
This one?
`self.gyro_scale = gyro_range * PiksiMulti.kGyroPrescale`

Comment (rikba) [reply piksi_multi_rtk_ros/src/piksi_multi_rtk_ros/piksi_multi.py:1567]: ```self.has_imu_scale = True``` is currently set twice. Once in [line 1567](https://github.com/ethz-asl/ethz_piksi_ros/pull/182/files#diff-e9d298b4dd316d49d65e3f18e24f331dR1567), once in [line 1570](https://github.com/ethz-asl/ethz_piksi_ros/pull/182/files#diff-e9d298b4dd316d49d65e3f18e24f331dR1570). Please remove it from line 1567.

**Review by rikba — APPROVED** [review]
LGTM. Thank you! :+1:

**Pull request merged**